L1Trigger/TrackerDTC/test/testDAQ_cfg.py

```diff
@@ -0,0 +1,66 @@
+################################################################################################
```

Comment:
Given that the DTC processes stubs, not clusters, why is there code to study cluster occupancy, but not stub occupancy?

Reply:
code that study stub occupancy is located here: L1Trigger/TrackerDTC/test/test_cfg.py

Reply:
OK. Please add comment to top of TrackerDTC/test_cfg.py and test_DAQ_cfg.py explaining that first runs DTC stub emulation, plots performance & stub occupancy, and second plots cluster occupancy.

Reply:
done